flybywiresim/a32nx
# feat: Added "Reset to Defaults" button to EFB throttle calibration page

Fixes #6113 

## Summary of Changes
This PR adds a button to reset the current throttle calibration values to the default values.

## Screenshots (if necessary)
![image](https://user-images.githubusercontent.com/16833201/146772521-106b55c2-f37f-4b40-bfbf-5968f77a9ecf.png)

## Additional context
This PR adds a new event A32NX_THROTTLE_MAPPING_SET_DEFAULTS and and new function in ThrottleAxisMapping.h/cpp to reset the current values for the calibration to their default values as defined in 

```
ThrottleAxisMapping::Configuration ThrottleAxisMapping::getDefaultConfiguration() {
  return {
      true,  // use reverse on axis
      -1.00,  // reverse low
      -0.95,  // reverse high
      -0.72,  // reverse idle low
      -0.62,  // reverse idle high
      -0.50,  // idle low
      -0.40,  // idle high
      -0.03,  // climb low
      +0.07,  // climb high
      +0.42,  // flex/mct low
      +0.52,  // flex/mct high
      +0.95,  // toga low
      +1.00   // toga high
  };
}
```

After resetting the user can start over with calibration and safe the calibration with the Safe button as before. 

Discord username (if different from GitHub): Cdr_Maverick#6475

## Testing instructions
1. Open EFB Throttle Calibration page - confirm new "Reset to Defaults" button is there and active
2. Change your config - mess it up - and use the button to reset the config to default values (see values above)
3. Apply & Save and check that the configuration is saved - also by restart the flight and re-starting the sim

Be aware the "Independent Axis" is not part of the saved configuration. This will not reset with the defaults. 


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
